packages/office-ui-fabric-react/src/components/Divider/examples/VerticalDivider.Custom.Example.tsx

```diff
@@ -45,9 +46,11 @@ export class VerticalDividerCustomExample extends React.Component<any, any> {
     const exampleClassNames = getExampleClassNames();
     return (
       <div className={exampleClassNames.wrapper}>
-        <p className={exampleClassNames.text}> Some text before the divider. </p>
-        <VerticalDivider getClassNames={getVerticalDividerClassNames} />
-        <p className={exampleClassNames.text}>Some text after the divider. </p>
+        <Customizer settings={{ theme: { palette: { themePrimary: 'pink' } } }}>
```

Comment:
Just a nit: 😊 
Not sure that for the sake of the issue resolution we wanna include the `Customizer` to demo the new `styled` approach. The `Customizer` should be demoed on its own page and relates more to theming. It should be enough to let the issue author know that it can now pick up the context passed with `Customizer`.

Reply:
Sounds good! I'll revert this one.